benchmark/runner/mini_runners.cpp

```diff
@@ -55,11 +77,21 @@ catalog::db_oid_t db_oid{0};
  */
 int64_t warmup_iterations_num{5};
 
+/**
+ * Should skip large rows iterations
+ */
+bool skip_large_rows = false;
```

Comment:
I believe this is to skip the rerun iteration instead of the warmup iteration? Please make the document more specific.

Reply:
Updated the documentation. This flag is actually used for only running up to the warmup limit (i.e TPCC).